cmd/appliance/extract_logs.go

```diff
@@ -58,6 +59,11 @@ func processJournalFile(file string, path string) error {
 	}
 	defer r.Close()
 
+	r.RegisterDecompressor(93, func(in io.Reader) io.ReadCloser {
```

Comment:
What's 93? This should use a constant or at least a comment.

Reply:
It's the number that they assigned in the zip format to the zstd algorithm.

```
In [2]: zipfile.ZIP_ZSTANDARD
Out[2]: 93
```

I will change it